oval/redhat/redhat.go

```diff
@@ -21,46 +21,60 @@ import (
 const (
 	ovalDir   = "oval"
 	redhatDir = "redhat"
-	urlFormat = "https://www.redhat.com/security/data/oval/v2/RHEL%s/rhel-%s.oval.xml.bz2"
-	retry     = 5
+
+	classVulnerability = "vulnerability"
+
+	urlFormat                   = "https://www.redhat.com/security/data/oval/v2/RHEL%s/rhel-%s.oval.xml.bz2"
+	urlFormatIncludingUnpatched = "https://www.redhat.com/security/data/oval/v2/beta/RHEL%s/rhel-%s-including-unpatched.oval.xml.bz2"
```

Comment:
They seem to already have including-unpatched under RHEL 6,7,8, not `beta/`. Also, we have to figure out the difference between rhel-6-extras-including-unpatched.oval.xml.bz2 and rhel-6-including-unpatched.oval.xml.bz2.